packages/shorebird_cli/lib/src/commands/patch/patch_android_command.dart

```diff
@@ -300,10 +305,24 @@ Looked in:
               releaseArtifactPath: releaseArtifactPath.value,
               patchArtifactPath: patchArtifactPath,
             );
+
+            String? hashSignature;
+            if (codeSignPrivateKey != null) {
+              final result = Process.runSync(
```

Comment:
Needs to investigate why this is returning empty, it works when ran in a shell. Or maybe should just go ahead and use a dart code to do the signing 🤷‍♂️